CHANGELOG.md

```diff
@@ -2,6 +2,82 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 54 end 2017-02-13
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+54+Ending+Feb+13%2C+2017%22+label%3Aenhancement)
+
+- Automate
+  - Alerts: Add hide & show alert status actions (backend) [(#13650)](https://github.com/ManageIQ/manageiq/pull/13650)
+  - Provisioning
+    - Change vLan name to Virtual Network  [(#13747)](https://github.com/ManageIQ/manageiq/pull/13747)
+    - Advanced networking placement features and automate exposure for OpenStack  [(#13608)](https://github.com/ManageIQ/manageiq/pull/13608)
```

Comment:
Should include PR https://github.com/ManageIQ/manageiq/pull/13853 which adds `set_security_groups` as an available method to automate during provisioning.